code/game/machinery/engi_points_vendor.dm

```diff
@@ -452,6 +452,7 @@
 	var/stop = 0
 	var/beat = 7
 	var/duration = 600
+	var/list/available = list()
 	var/static/list/choices = list("Engineering's Basic Beat", "Engineering's Domination Dance", "Engineering's Superiority Shimmy", "Engineering's Ultimate High-Energy Hustle")

```

Comment:
Ok so:
```tracks["Engineering's Basic Beat"]``` points to the first list, the list has been *keyed* to the string ```"Engineering's Basic Beat"```, the same is true for the second and third and Nth lists, all *keyed* to the second, third and Nth keys.

once you have access to the list, by getting it from a key and putting it in a var
```var/list/trackinfo = tracks["Engineering's Basic Beat"]``` then you can access that variable as normal, with indices ```trackinfo[1]``` (filepath) ```trackinfo[2]``` (duration)

make sense?